tests/requirements.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
I would just leave all the `requirements` at the base. That seems to be common for pip-based python projects. It makes it immediately obvious what it is. I did not move the one in crawler to the base because I didn't want to conflict with your existing PR.